src/backend/app.js

```diff
@@ -19,16 +20,69 @@ app.use(express.json());
 
 app.use(cors());
 
-router.use("/meals", mealsRouter);
+router.use('/meals', mealsRouter);
+
+const errMessage = 'Sorry, there are no meals.';
+
+app.get('/future-meals', async (req, res) => {
+  const [row] = await knex.raw(
+    ' SELECT * FROM `meal` WHERE `when` > "2022-08-11"'
```

Comment:
Instead of hardcoding the date, you can use JavaScript Date methods for now.

Reply:
Thank you for your review.
I agree with you and will fix it.